mkdocs.yml

```diff
@@ -31,6 +31,7 @@ nav:
     - Purchase Filtering: 'tutorials/purchase-filtering/tutorial.md'
   - Producing Data to Kafka: producer.md
   - Processing & Transforming Data: processing.md
+  - GroupBy Operation (re-keying): groupby.md
```

Comment:
Suggestion: rename to "Grouping Data by Keys". The word "re-keying" may not be familiar to everyone, and it's actually explained within the doc

Reply:
I thought about something similar, though I also wanted to include  `GroupBy` specifically because it's a well known term.

Wanted the section to jump out to someone who has no idea what re-keying is, but does know what GroupBy is, whereas "Grouping Data by Keys" may not jump out to that person.

Maybe theres a middle ground (though maybe a bit verbose) with something like "GroupBy (grouping data by keys)"?

Reply:
Let's simply do "GroupBy Opeartion". 
I'd like to avoid adding anything in `()`. The title should be concise